raytracing/imagingpath.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, elements=None, label=""):
         self.showPointsOfInterest = True
         self.showPointsOfInterestLabels = True
         self.showPlanesAcrossPointsOfInterest = True
+        self.images = None
```

Comment:
Recurring comment here: I think to keep the nomenclature standard, we should refer to these as something like `intermediateForwardConjugates` or `intermediateForwardConjugatePlanes` or maybe just `intermediateConjugates`. because the class `Matrix` already has `forwardConjugate` and `backwardConjugate` to find the last conjugate plane after or the first before the element.  `Matrix` has now internal structure, so it makes no sense to have `self.intermediateForwardConjugates` functions there.  However, `MatrixGroup` is the most general class after `Matrix` that has an internal structure , so we should probably move that function there (later?). A `MatrixGroup` does have intermediate conjugate planes, whether it is an imaging path or not.
Also, these planes exist regardless of what the object is (present, absent, etc...), so we should refer to them as `planes` more than `images`.

That being said, we should remove the intermediate variable that only adds complexity and chances of error (see below at `computeImages()`)